Criados os documentos que descrevem o formato dos eventos do GOJS

jlcarvalho opened the pull request:
A ideia desse PR é criar ferramentas que facilitem a organização dos eventos do GOJS, pois quanto mais fácil for organizar, mais fácil será passar o bastão e mais eventos com maior qualidade nós teremos.

Grande parte do conteúdo desse PR é baseado nos guias do FEMUG, que foram construídos com anos de experiência na organização de meet-ups, porém adaptados a nossa realidade goiana.

Esse PR ainda não está pronto para ser mergeado, ainda faltam criar alguns templates, guias na wiki, checklists, etc.. Porém acredito que já está maduro o suficiente para discutirmos a respeito.

jlcarvalho committed:
Criados os documentos que descrevem o formato dos eventos do GOJS

omarciovsena commented on README.md:
```diff
@@ -1,11 +1,125 @@
+
+Independentemente do formato escolhido para o evento, é importante que o mesmo seja agendado e divulgado com antecedência. Anunciar e divulgar o evento de 3 a 4 semanas de antecedência dará tempo para as pessoas se organizarem para comparecer ao mesmo.
+
+Também recomendamos ser claro claro sobre qual audiência o facilitador está buscando atingir com o encontro. Por exemplo, o evento é um tema específico? Em iniciantes? etc..
```
palavras duplicadas: `claro claro`

omarciovsena commented on README.md:
```diff
@@ -1,11 +1,125 @@
+
+Também fornecemos dois modelos de imagem que você pode usar para criar banners digitais para promover o evento.
+
+![img1]()
```
Faltam as imagens :P

jlcarvalho replied:
Tá dentro das coisas que vou providenciar antes do PR ficar pronto pra merge 😛

omarciovsena commented on README.md:
```diff
@@ -1,11 +1,125 @@
+## Formatos de eventos
+- **Meet-ups com palestras**, são os eventos mais comuns da comunidade e recomendamos que sejam realizados mensalmente. Normalmente possuem duração de 3 horas com uma recepção/networking de 30 minutos, 2 ou 3 slots de tempo com cerca de 20 minutos cada para apresentação de palestras. Ficando o tempo restante mais networking.
+- **Encontros para networking**, são eventos esporádicos que tem como objetivo permitir que os membros da comunidade se conheçam. Esses são os eventos mais versáteis pois podem acontecer em qualquer lugar (bares, shoppings, auditórios, etc.). O importante é que os participantes tenham espaço para conversar e trocar experiências.
+- **Hackathons**, são eventos onde a comunidade se reúne (presencial ou remotamente) para criar projetos em torno de um tema específico. Atualmente não temos um modelo para seguir para esse tipo de evento, porém podemos utilizar o [The Hack Day Manifesto](https://hackdaymanifesto.com/) e o [Hackathon Guide](https://hackathon.guide/) como guias.
```
`presencial ou remoto` não seria mais equivalentes?

jlcarvalho replied:
Sim, vou ajustar.

jlcarvalho committed:
Corrige typos

brunocroh committed:
Adiciona uma simples descrição para a comunidade

Quando vamos criar/divulgar um evento é legar que tenhamos um pequeno texto falando um pouco do que é o GOjs, adicionei um breve esboço, inspirado em um texto antigo de divulgação de evento do @jpmsegurado, que podemos ir evoluindo e melhorando com o tempo.
